app/kubernetes/services/resourcePoolService.js

```diff
@@ -114,6 +121,10 @@ export function KubernetesResourcePoolService($async, EndpointService, Kubernete
       await KubernetesNamespaceService.delete(pool.Namespace);
     });
   }
+
+  function toggleSystem(endpointId, namespaceName, system) {
```

Comment:
hmm, does this need to update the store?

Reply:
Nop, store will update anyway on refresh/reload when toggle ends as we have `this.$state.reload();`